deployment-dapp/src/publishSellOrderScript.ts

```diff
@@ -70,23 +74,23 @@ const main = async () => {
   if (!appAddress) throw Error('Failed to get app address'); // If the app was not deployed, do not continue
 
   // validate params
-  params().validate(PRICE);
-  params().validate(VOLUME);
+  const price: Params = await positiveNumberSchema()
+    .default(DEFAULT_APP_PRICE)
+    .label('PRICE')
+    .validate(PRICE);
```

Comment:
remove useless and incorrect type (here `validate()` returns `Promise<number>`)
```suggestion
  const price = await positiveNumberSchema()
    .default(DEFAULT_APP_PRICE)
    .label('PRICE')
    .validate(PRICE);
```